packages/inter-protocol/src/proposals/econ-behaviors.js

```diff
@@ -69,6 +69,7 @@ export const SECONDS_PER_WEEK = 7n * SECONDS_PER_DAY;
  *     import('../vaultFactory/vaultFactory.js')['start']
  *   >;
  *   auctioneerKit: AuctioneerKit;
+ *   newAuctioneerKit: AuctioneerKit;
```

Comment:
will this always be here? I wonder if we need a convention for replacement. e.g. `nextAuctioneerKit` or `stagedAuctioneerKit`. I think it should be optional so that we remove it from bootstrap space after it becomes the canonical one.

Reply:
any string is a valid name in the space. you could `reset` its promise so that there's no associated value. And maybe change the typedef to `never` so people get static analysis feedback not to use it.

Reply:
#8757 has been replaced with #9283. It has those clean-ups.